### DISCUSSION TITLE
Add custom scrape targets to Prometheus Operator

### DISCUSSION THREAD

**Discussion by hemant659** [prometheus-operator/prometheus-operator#4578] [Q&A] [answered]
I want to monitor my applications which are running on AWS ec2-instances and are outside the Kubernetes cluster(EKS).  Please suggest a way I can add these ec2 machines as targets in Prometheus configuration in my Prometheus Operator setup.

**Comment by simonpasquier** [comment, marked as answer]
You should probably use `spec.additionalScrapeConfigs`.

**Reply by hemant659** [reply]
Thanks, it worked
